src/disco/bank/fd_txncache.c

```diff
@@ -349,33 +352,9 @@ fd_txncache_delete( void * shtc ) {
 static void
 fd_txncache_remove_blockcache_idx( fd_txncache_t * tc,
                                    ulong idx ) {
-  tc->blockcache[ idx ].lowest_slot = ULONG_MAX;
+  tc->blockcache[ idx ].lowest_slot = ULONG_MAX - 1UL;
```

Comment:
need FD_TXNCACHE_XXX constants for the `(ULONG_MAX - 1UL)`, `-2UL` business that describe what's going on and documenting that 

i won't block this PR on that but please add

Reply:
@arjain4 can you do this in your slotcache PR?

Reply:
yep have those changes in mind